tests/datasource/test_pandas_datasource.py

```diff
@@ -127,64 +127,6 @@ def test_create_pandas_datasource(
     )
 
 
-@pytest.mark.filesystem
```

Comment:
Do you mind summarizing why these tests are getting deleted? I'm assuming it's for a good reason, but from test names alone it's not clear how they are related to the work at hand.

Reply:
sure! the deleted tests exercise deprecated logic which we're about to remove, so i decided are probably not worth the energy to fix. Most use the `batch_kwargs` api and custom expectations.